lib/wallet/src/Cardano/Wallet/Write/Tx.hs

```diff
@@ -867,6 +868,12 @@ getFeePerByte era pp = FeePerByte $ case era of
     RecentEraConway -> pp ^. #_minfeeA
     RecentEraBabbage -> pp ^. #_minfeeA
 
+feeOfBytes
+    :: FeePerByte
+    -> Natural
```

Comment:
TxSize?

Reply:
I agree that it would be nice to use something more self-documenting than `Natural` here.

Perhaps `TxSize` or `TxSizeBytes` would be good.

Reply:
I'd like to avoid importing `TxSize` from `Write.Tx`, but if we ever end up with some dedicated `.Size` or `.Fees` module it might come more natural. Leaving for now.